### ISSUE TITLE
Search facets weird display on mobile

### ISSUE THREAD

**Issue by sfisher** [datadryad/dryad-product-roadmap#1545]
This now seems to display mostly OK after some geoblacklight upgrades.

- Only seems to display up to 3 facets under a menu in mobile.  Is there a way to configure to show them all?

To me this seems very low priority since I don't think a lot of people are searching the Dryad site from our own search and I understand that the majority of people come to the site from other locations rather than using our search.

**Comment by ahamelers** [comment]
I think right now the search facets aren't even displaying on mobile! Definitely needs a redesign. Waiting on the removal of geoblacklight to be done.

**Comment by ahamelers** [comment]
Closing in favor of #3917
